install-dev/langs/en/data/tab.xml

```diff
@@ -133,4 +134,7 @@
     <tab id="Zones" name="Zones" />
     <tab id="Marketing_tools" name="Module Selections" />
     <tab id="Tabs" name="Menu" />
+    <tab id="Security_1" name="Security" />
```

Comment:
`Security_1` ?

Reply:
Because the id is already used in another tab.xml file.